explorer/site-config/anvil-cmg/dev/config.ts

```diff
@@ -18,19 +17,12 @@ import { filesEntityConfig } from "./index/filesEntityConfig";
 import { summary } from "./index/summary";
 
 // Template constants
-const BROWSER_URL = "https://anvil.gi.ucsc.edu";
 const SLOGAN = "NHGRI Analysis Visualization and Informatics Lab-space";
 export const URL_DATASETS = "/datasets";
-const LOGO: LogoProps = {
-  alt: SLOGAN,
-  height: 40,
-  link: "/",
-  src: logoAnvil,
-};
 
-const config: SiteConfig = {
+export function make_config(browserUrl: string): SiteConfig { return {
```

Comment:
So that the browser URL becomes configurable without duplicating large amounts of config.